Sources/Build/BuildManifest/LLBuildManifestBuilder.swift

```diff
@@ -322,7 +322,7 @@ extension ResolvedModule {
     }
 
     public func getLLBuildTargetName(buildParameters: BuildParameters) -> String {
-        "\(self.name)-\(buildParameters.buildConfig)\(buildParameters.suffix(triple: self.buildTriple)).module"
+        "\(self.name)-\(buildParameters.triple.tripleString)-\(buildParameters.buildConfig)\(buildParameters.suffix(triple: self.buildTriple)).module"
```

Comment:
What about other names below, do we not need exe and libs associated with triples as well?

Reply:
👍🏻

Reply:
Yes, we need it to leverage build cache more when switching triples